mitxpro/settings.py

```diff
@@ -216,19 +217,20 @@
     "mitol.digitalcredentials.apps.DigitalCredentialsApp",
     "mitol.mail.apps.MailApp",
     "mitol.oauth_toolkit_extensions.apps.OAuthToolkitExtensionsApp",
+    "mitol.authentication.apps.TransitionalAuthenticationApp",
 )
 # Only include the seed data app if this isn't running in prod
 if ENVIRONMENT not in ("production", "prod"):
     INSTALLED_APPS += ("localdev.seed",)
 
 
-DISABLE_WEBPACK_LOADER_STATS = get_bool(
+DISABLE_WEBPACK_LOADER_STATS = WEBPACK_DISABLE_LOADER_STATS = get_bool(
```

Comment:
I had a thought that we should be using only`WEBPACK_DISABLE_LOADER_STATS` but I had a discussion with @Wassaf-Shahzad and it turns out that making this naming standard would be part of `ol-django` otherwise it might require some major changes while trying to keep only one variable.

Reply:
You can actually remove all these webpack settings definitions altogether and `from mitol.common.settings.webpack import *` at the top